inspirehep/modules/workflows/templates/inspire_workflows/index.html

```diff
@@ -77,13 +74,34 @@ <h3 class="text-center header-text">
 
       <holding-pen-dashboard-item
         template="{{url_for('static', filename='js/inspire_workflows_ui/templates/dashboard_stat_block.html')}}"
-        primary_filter_key="source"
-        primary_filter_value="arXiv"
+        filter_string="source=arXiv"
         secondary_filter="status"
         section_title="arxiv harvests">
       </holding-pen-dashboard-item>
 
     </div>
+
+    <div class="col-md-3 col-sm-6">
+
+      <holding-pen-dashboard-item
+        template="{{url_for('static', filename='js/inspire_workflows_ui/templates/dashboard_stat_block.html')}}"
+        filter_string="source=Alpha experiment"
```

Comment:
It's like this now, so as separate dashboard items, so that you can choose which blocks to add, this is as configurable as it gets for now. 
The submission area below will hold all stats however. I will get around to adding this ASAP. I already have the data, just need to write the visualization.